mlflow/store/model_registry/sqlalchemy_store.py

```diff
@@ -100,11 +102,12 @@ def _save_to_db(self, session, objs):
             # single object
             session.add(objs)
 
-    def create_registered_model(self, name):
+    def create_registered_model(self, name, creation_time=None):
```

Comment:
I chose this approach because of its similarity to how we account for flakiness when testing search_runs pagination: https://github.com/mlflow/mlflow/blob/HEAD/tests/store/tracking/test_sqlalchemy_store.py#L1381